### PR TITLE
Made robot.js use inches and scaled everything by 3 in base.js

### PR DESCRIPTION
**Pull request by charleshong3**
3 pixels = 1 inch
Robot size = 60x80 pixels = 20x26.7 inches
Field size = 432x432 pixels = 144x144 inches

### CODE REVIEW

File: static/js/base.js [hunk 1]
```diff
@@ -37,12 +37,15 @@ function update(state) {
...
+    let y = state.Y * scaleFactor;
+    let dir = state.dir;
     document.getElementById("demo").innerHTML = state.X.toFixed(2) + ", " + state.Y.toFixed(2)
     var robotRect = document.querySelector("rect")
-    robotRect.setAttributeNS(null, "x", state.X)
-    robotRect.setAttributeNS(null, "y", state.Y)
-    var rotateStr = "rotate(" + state.dir + " " + (state.X + 15*scaleFactor) + " " + (state.Y + 20*scaleFactor) + ")"
+    robotRect.setAttributeNS(null, "x", x)
+    robotRect.setAttributeNS(null, "y", y)
+    var rotateStr = "rotate(" + dir + " " + (x + 30) + " " + (y + 40) + ")"
```
Comment (baby-bell) [inline static/js/base.js]:
```suggestion
    let rotateStr = `rotate({dir} {x + 30} {y + 40})`
```

File: static/js/base.js [hunk 2]
```diff
@@ -37,12 +37,15 @@ function update(state) {
     Update the state (position and direction) of the robot.
     Example of state: {x:72, y:72, dir:0}
     */
-    var scaleFactor = 2;
+    var scaleFactor = 3;
+    let x = state.X * scaleFactor;
```
Comment (baby-bell) [inline static/js/base.js]:
```suggestion
    const scaledX = state.X * scaleFactor;
    const scaledY = state.Y * scaleFactor;
```

**Review by baby-bell — DISMISSED** [review]
just in general, use "const" for things that will not change, and "let" in place of "var".

**Review by kevinzhu00 — APPROVED** [review]
Changes have been fixed.

**Pull request merged**